cmd/get.go

```diff
@@ -13,6 +13,32 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type outputEnum string
+
+const (
+	outputStandard outputEnum = "standard"
+	outputNotes    outputEnum = "notes"
+)
+
+func (e *outputEnum) String() string {
+	return string(*e)
+}
+
+func (e *outputEnum) Set(v string) error {
+	switch v {
+	case string(outputStandard), string(outputNotes):
+		*e = outputEnum(v)
+		return nil
+	default:
+		return fmt.Errorf(`must be one of %s or %s`, outputStandard, outputNotes)
+	}
+}
+
+func (e *outputEnum) Type() string {
+	return "outputEnum"
+}
```

Comment:
How about we keep the implementation inside the writer package?

I'll explain what I mean over the next few comments.

Reply:
This is only for handling the command line arguments. Open to suggestions on how to remove some of the duplication.